pkg/controller/install/errors.go

```diff
@@ -42,6 +42,15 @@ func IsErrorUnrecoverable(err error) bool {
 	return ok
 }
 
+// IsErrorDeploymentUpdated reports if a given strategy error is due to an update to the deployment.
```

Comment:
what's the point of the function if all input is going to have error? Could we just move the reasonForError statement to where it is used?

Reply:
Sure - that sounds good to me.